main.tf

```diff
@@ -3015,7 +3015,7 @@ module "karpenter" {
   namespace        = local.karpenter_namespace
   create_namespace = try(var.karpenter.create_namespace, true)
   chart            = try(var.karpenter.chart, "karpenter")
-  chart_version    = try(var.karpenter.chart_version, "0.35.0")
+  chart_version    = try(var.karpenter.chart_version, "v0.32.1")
```

Comment:
why the downgrade?
v0.35.1 is out

Reply:
I think this issue still exists and needs fixing https://github.com/aws/karpenter-provider-aws/issues/5415

Reply:
A workaround could be pinning the image tag to the correct one instead perhaps but I haven't tested that

Reply:
thank you, but I am unable to reproduce the error - the default version is just that, a default. we encourage users to specify the version in order to have better control over which version is used and when that changes. for now, I would say set the v0.32.1 version in your configuration if that what puts things back into a healthy state. Closing this out for now though

Reply:
Interesting you couldn't reproduce with the default values. Did you test this with an upgrade or in a fresh install?

I'm happy for it to be closed if no one else is seeing the issue, perhaps it's related to our deployment but we're installing karpenter with mostly default values including the image and helm chart